### ISSUE TITLE
AttributeError at /de/admin/cms/page/5/de/publish/ 'Text' object has no attribute '_placeholder_cache'

### ISSUE THREAD

**Issue by jkoerkel** [aldryn/aldryn-newsblog#209]
Publishing pages with a text plugin not possible. aldryn-newsblog 0.9.4

```
Environment:


Request Method: GET
Request URL: http://127.0.0.1:8000/de/admin/cms/page/5/de/publish/

Django Version: 1.7.8
Python Version: 2.7.9
Installed Applications:
('djangocms_admin_style',
 'filer',
 'aldryn_apphooks_config',
 'aldryn_boilerplates',
 'aldryn_categories',
 'aldryn_newsblog',
 'aldryn_people',
 'aldryn_reversion',
 'djangocms_text_ckeditor',
 'easy_thumbnails',
 'parler',
 'reversion',
 'sortedm2m',
 'taggit',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.admin',
 'django.contrib.sites',
 'django.contrib.sitemaps',
 'django.contrib.staticfiles',
 'django.contrib.messages',
 'cms',
 'menus',
 'sekizai',
 'treebeard',
 'djangocms_style',
 'djangocms_column',
 'djangocms_file',
 'djangocms_flash',
 'djangocms_googlemap',
 'djangocms_inherit',
 'djangocms_link',
 'djangocms_picture',
 'djangocms_teaser',
 'djangocms_video',
 'juliankoerkel_de')
Installed Middleware:
('django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.locale.LocaleMiddleware',
 'django.middleware.doc.XViewMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'cms.middleware.user.CurrentUserMiddleware',
 'cms.middleware.page.CurrentPageMiddleware',
 'cms.middleware.toolbar.ToolbarMiddleware',
 'cms.middleware.language.LanguageCookieMiddleware')


Traceback:
File "/home/julian/projects/jk/local/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
  111.                     response = wrapped_callback(request, *callback_args, **callback_kwargs)
File "/home/julian/projects/jk/local/lib/python2.7/site-packages/django/utils/decorators.py" in _wrapped_view
  105.                     response = view_func(request, *args, **kwargs)
File "/home/julian/projects/jk/local/lib/python2.7/site-packages/django/views/decorators/cache.py" in _wrapped_view_func
  52.         response = view_func(request, *args, **kwargs)
File "/home/julian/projects/jk/local/lib/python2.7/site-packages/django/contrib/admin/sites.py" in inner
  206.             return view(request, *args, **kwargs)
File "/home/julian/projects/jk/local/lib/python2.7/site-packages/django/db/transaction.py" in inner
  394.                 return func(*args, **kwargs)
File "/home/julian/projects/jk/local/lib/python2.7/site-packages/reversion/revisions.py" in do_revision_context
  297.                 return func(*args, **kwargs)
File "/home/julian/projects/jk/local/lib/python2.7/site-packages/cms/admin/pageadmin.py" in publish_page
  956.             published = page.publish(language)
File "/home/julian/projects/jk/local/lib/python2.7/site-packages/cms/models/pagemodel.py" in publish
  608.             self._copy_contents(public_page, language)
File "/home/julian/projects/jk/local/lib/python2.7/site-packages/cms/models/pagemodel.py" in _copy_contents
  308.                 copy_plugins_to(plugins, ph, no_signals=True)
File "/home/julian/projects/jk/local/lib/python2.7/site-packages/cms/utils/copy_plugins.py" in copy_plugins_to
  32.             new_instance.post_copy(old_plugin, plugins_ziplist)
File "/home/julian/projects/jk/local/lib/python2.7/site-packages/djangocms_text_ckeditor/models.py" in post_copy
  73.         self.save()
File "/home/julian/projects/jk/local/lib/python2.7/site-packages/djangocms_text_ckeditor/models.py" in save
  44.         super(AbstractText, self).save(*args, **kwargs)
File "/home/julian/projects/jk/local/lib/python2.7/site-packages/cms/models/pluginmodel.py" in save
  238.         super(CMSPlugin, self).save()
File "/home/julian/projects/jk/local/lib/python2.7/site-packages/django/db/models/base.py" in save
  589.                        force_update=force_update, update_fields=update_fields)
File "/home/julian/projects/jk/local/lib/python2.7/site-packages/django/db/models/base.py" in save_base
  626.                                    update_fields=update_fields, raw=raw, using=using)
File "/home/julian/projects/jk/local/lib/python2.7/site-packages/django/dispatch/dispatcher.py" in send
  198.             response = receiver(signal=self, sender=sender, **named)
File "/home/julian/projects/jk/local/lib/python2.7/site-packages/aldryn_newsblog/models.py" in update_seach_index
  523.         placeholder = instance._placeholder_cache

Exception Type: AttributeError at /de/admin/cms/page/5/de/publish/
Exception Value: 'Text' object has no attribute '_placeholder_cache'
```

**Comment by alexcustos** [comment]
I've got exactly the same problem with 0.9.5. It looks like there is a raise condition in this place. If I stop the debugger at "placeholder = instance._placeholder_cache" when the attribute is not there, then waiting for a few seconds before execute it, all works fine. But I facing the problem in real-time.

**Comment by alexcustos** [comment]
I've got exactly the same problem with 0.9.5. It's very strange issue. When I apply this patch all works fine, but it's a very poor workaround:
```
+ if not hasattr(instance, '_placeholder_cache'):
+     import inspect
+     inspect.getmembers(instance)
placeholder = instance._placeholder_cache
```

**Comment by mkoistinen** [comment]
Closed by #220.
